config.yml.erb

```diff
@@ -630,3 +630,7 @@ openai_evaluate_rubric_org_id:
 
 # AI Proxy for OpenAI API
 ai_proxy_origin: https://aiproxy-test.code.org
+
+# Statsig keys
+statsig_server_secret_key: !Secret
```

Comment:
We want to use StatSig `local_mode` on ALL environments except our managed test server (`test.code.org / test-studio.code.org`) and production. It’s probably simpler to declare the fake secrets in the parent `config.yml.erb` file and then override in just the `test.yml.erb` and `production.yml.erb`.

Less important … to help others recognize that the key/secret are fake for testing, I suggest using values that clearly indicate that, like we do in similar cases https://github.com/code-dot-org/code-dot-org/blob/ed073c68cd44c34023f05327d11f17b2f07d39ce/config/development.yml.erb#L40-L41

Reply:
Ah yes, refactoring this to default to fake secrets in config.yml.erb and override in test and production makes sense. Will make this change.

I added a comment above the fake secret sections in each config file. One of the issues is that Statsig expects the secret to be prefixed with `secret-`. I could do something like `secret-fake-key`. I've tested this locally and Statsig doesn't error on init with this fake key.

Reply:
In `test.yml.erb`, we’ll need something like this:

``` erb
# The only type of test environment we enable StatSig is the managed test server (`test.code.org / test-studio.code.org`)
# and only in Ruby code executing within the web application server.
<% if running_web_application? && test_system? -%>
statsig_server_secret_key: !Secret
statsig_api_client_key: !Secret
<% else %>
statsig_server_secret_key: 'fake-secret-key'
statsig_api_client_key: 'fake-client-key'
<% end -%>
```

Reply:
> I added a comment above the fake secret sections in each config file. One of the issues is that Statsig expects the secret to be prefixed with `secret-`. I could do something like `secret-fake-key`. I've tested this locally and Statsig doesn't error on init with this fake key.

Apologies - you took the time to document the prefix requirement in the Pull Request, and I didn’t read closely! 😞